scripts/sanity_chk/harness.py

```diff
@@ -121,6 +127,40 @@ def handle(self, line):
 
         self.process_test(line)
 
+class Pytest(Harness):
+    def handle(self, line):
+        cmd = 'pytest -s ' +  self.handle_script + \
+            ' --cmdopt=' + self.running_dir + ' --junit-xml=' + \
+                self.running_dir + '/report.html'
+
+        with subprocess.Popen(cmd, shell=True) as proc:
+            try:
+                proc.wait(15)
+                tree = ET.parse(self.running_dir + "/report.html")
```

Comment:
Verify you have the file before opening it.

Reply:
fixed, used IOError exception to handle this.